doc/newsfragments/report_success_for_empty_run.rst

```diff
@@ -0,0 +1 @@
+* Testplan will report success in case of an empty run
```

Comment:
End announcement entry with a period please. (Not that it's better, just for consistency)

Reply:
Done